gitlab-ci-helper.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
```

Comment:
file should be called .bash if it uses bash.
when possible, try to use posix sh and shellcheck to check for errors